R/add_ressource_path.R

```diff
@@ -0,0 +1,20 @@
+#' Add resource path
+#'
+#' @param prefix The URL prefix (without slashes). Valid characters are a-z, A-Z, 0-9, hyphen, period, and underscore. For example, a value of 'foo' means that any request paths that begin with '/foo' will be mapped to the given directory.
+#' @param directory_path The directory that contains the static resources to be served.
+#'
+#' @importFrom rlang is_empty
+#' @importFrom shiny addResourcePath
+#' 
+#' @export
+#'
+add_resource_path <- function(prefix, directory_path){
```

Comment:
I think the message should be disabled by default, but with an argument that allow to show it if needed. 

Something like `add_resource_path(..., warn = FALSE)`